client/src/elm/Translate.elm

```diff
@@ -15303,202 +15303,202 @@ translationSet trans =
+            , kinyarwanda = Just "Intego y'ubu butumwa ni ukugufasha nk'umukozi utanga serivisi z'ubuvuzi kumva umeze neza kandi wifitemo icyizere n'umutekano mu kazi ukora ndetse n'ubuzima bwawe bwa buri munsi."
             }
 
         ResilienceMessageIntroduction2Title ->
             { english = "More information about the program"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Andi makuru ku byerekeye iyi gahunda"
             }
 
         ResilienceMessageIntroduction2Paragraph1 ->
             { english = "There are several things we face at work such as being given too many responsibilities and working overtime. However, we can at least learn how to deal with extreme fatigue and stress."
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Hari ibintu byinshi duhura nabyo mu kazi nko guhabwa inshingano nyinshi, gukora amasaha y'ikirenga,... Ariko byibuze dushobora kwiga uko twarwanya umunaniro ukabije."
             }
 
         ResilienceMessageIntroduction2Paragraph2 ->
             { english = "After this six-month journey, we hope that you will be able to"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Nyuma y'uru rugendo rw'amezi atandatu, twizera ko uzaba ushobora:"
             }
 
         ResilienceMessageIntroduction2Bullet1 ->
             { english = "Know what to do when you are under extreme fatigue or stress"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Kumenya uko witwara mu gihe ufite umunaniro ukabije (stress)"
             }
 
         ResilienceMessageIntroduction2Bullet2 ->
             { english = "Recover after going through difficult times"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Kongera kumererwa neza nyuma yo kunyura mu bihe bigoye"
             }
 
         ResilienceMessageIntroduction2Bullet3 ->
             { english = "Find time and learns ways to relax"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Kubona akanya ko kuruhuka no kumenya uburyo bwo kuruhuka"
             }
 
         ResilienceMessageIntroduction2Bullet4 ->
-            { english = "Know how to get along well with those we live with and work with."
-            , kinyarwanda = Nothing
+            { english = "Know how to get along well with those we live with and work with"
+            , kinyarwanda = Just "Kumenya kubanira neza abo tubana ndeste nabo dukorana"
             }
 
         ResilienceMessageIntroduction2Bullet5 ->
             { english = "Find happiness and enjoy life"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Kunezerwa ndetse no kuryoherwa n'ubuzima"
             }
 
         ResilienceMessageIntroduction3Title ->
             { english = "Get a notebook ready!"
-            , kinyarwanda = Nothing
+            , kinyarwanda = Just "Tegura ikaye yo kujya wandikamo"
```

Comment:
Why no `!`?

Reply:
> Why no `!`?

@anvmn the way it was done on the some text they are just some sayings in english and they find a similar saying in Kinyarwanda but a translation of the English version